lib/landing_page.dart

```diff
@@ -77,8 +77,16 @@ class LandingPage extends StatelessWidget {
                     fontWeight: FontWeight.w200,
                   )),
               SizedBox(height: 80),
+              Text('Fortschritt',
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontFamily: 'Impact',
+                    color: Colors.white,
+                  )),
+
               Container(
```

Comment:
dieser container könnte ein issue seien. ich denke nicht das er nötig ist. die höhe sollte in progressBar selbst festgelegt werden